### PR TITLE
Add Vine Server.app 5.3.0

### PR DESCRIPTION
**Pull request by stweil**
Signed-off-by: Stefan Weil <sw@weilnetz.de>

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew install --cask {{cask_file}}` worked successfully.
- [x] `brew uninstall --cask {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by stweil** [comment]
This cask was removed two years ago in commit 80e35b757453f911470de4c99f69b5f09fd5c6d9. The new cask uses a different maintained source.

`brew audit` reports an error. It suggests to use `sha256 :no_check` which looks strange.

**Comment by miccal** [comment]
Thank you for your contribution to `homebrew-cask` @stweil.

Could you please try to add a [`zap` stanza](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md), thanks.

**Comment by stweil** [comment]
> Could you please try to add a zap stanza, thanks.

Done.

File: Casks/vine-server.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  depends_on macos: ">= :sierra"
+
+  app "Vine Server.app"
+  binary "#{appdir}/Vine Server.app/Contents/MacOS/OSXvnc-server"
+  binary "#{appdir}/Vine Server.app/Contents/MacOS/storepasswd"
+  binary "#{appdir}/Vine Server.app/Contents/MacOS/Vine Server"
+
+  zap trash: [
+    "~/Library/Preferences/de.uni-mannheim.VineServer.plist",
+    "/Library/Application Support/VineServer",
```
Comment (miccal) [inline Casks/vine-server.rb:19]: Just to confrim that this is `/Library/Application Support/VineServer`, and not `~/Library/Application Support/VineServer`?

Comment (stweil) [reply Casks/vine-server.rb:19]: That's correct.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**